src/Activation/ActivationSummary.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { FunctionComponent } from "react"
+import {
+  TouchableOpacity,
+  Linking,
+  ScrollView,
+  View,
+  StyleSheet,
+  Image,
+} from "react-native"
+import { useTranslation } from "react-i18next"
+
+import { usePermissionsContext } from "../PermissionsContext"
+import { useOnboardingContext } from "../OnboardingContext"
+import { useApplicationName } from "../More/useApplicationInfo"
+import { GlobalText } from "../components"
+import { Button } from "../components"
```

Comment:
```suggestion
import { Button, GlobalText } from "../components"
```

Reply:
Yeah I think @amandabeiner were thinking of tackling that tomorrow and building some kind of `ActivationContext` to handle all that stuff, plus the status of Bluetooth and Location.